TYPO3.Media/Classes/TYPO3/Media/Command/MediaCommandController.php

```diff
@@ -135,7 +135,7 @@ public function importResourcesCommand($simulate = false)
      * Additionally accepts a ``async`` parameter determining if the created thumbnails are generated when created.
      *
      * @param string $preset Preset name, if not provided thumbnails are created for all presets
-     * @param boolean $async Asynchronous generation, if not provided the setting ``TYPO3.Media.asyncThumbnails`` is used
+     * @param boolean $async Asynchronous generation, if not provided the setting ``Neos.Media.asyncThumbnails`` is used
```

Comment:
The help message will point to a non-existing setting (for now), so don't merge without actual vendor name change.